trunk/NDHMS/Routing/Reservoirs/RFC_Forecasts/module_rfc_forecasts.F

```diff
@@ -310,6 +311,19 @@ subroutine run_rfc_forecasts_reservoir(this, previous_timestep_inflow, inflow, &
             ! Set outflow to corresponding discharge from array
             this%output%outflow = this%state%discharges(this%state%time_series_index)
 
+            ! Update water elevation
+            this%state%water_elevation = this%state%water_elevation + &
+            ((this%input%inflow - this%output%outflow) / this%properties%lake_area) * routing_period
+
+            ! Ensure that the water elevation is within the minimum and maximum elevation
+            if (this%state%water_elevation < 0.0) then
+                this%state%water_elevation = 0.0
```

Comment:
Is water elevation height above lake bed or height above sea level? in the second case this may be invalid in a couple specific areas (Death Valley, New Orleans, Denmark)